From 09cb6d7b467f6d1c6685ee9ccc171f4dddbe1f42 Mon Sep 17 00:00:00 2001
From: Simon McVittie <simon.mcvittie@collabora.co.uk>
Date: Sat, 7 Nov 2015 13:03:47 +0100
Subject: [PATCH] Drop [Install] sections from user services

We install the symlink to enable dbus.socket statically, so it doesn't
make much sense to invoke `systemctl enable` on it; and
dbus.service should normally be started by socket activation
(or possibly an explicit dependency) rather than manually.

Bug: https://bugs.freedesktop.org/show_bug.cgi?id=92402
Signed-off-by: Simon McVittie <simon.mcvittie@collabora.co.uk>
Reviewed-by: Lennart Poettering
---
 bus/systemd-user/dbus.service.in | 3 ---
 bus/systemd-user/dbus.socket.in  | 4 ----
 2 files changed, 7 deletions(-)

diff --git a/bus/systemd-user/dbus.service.in b/bus/systemd-user/dbus.service.in
index b41f522b8af8c174..4355d72844079cc5 100644
--- a/bus/systemd-user/dbus.service.in
+++ b/bus/systemd-user/dbus.service.in
@@ -6,6 +6,3 @@ Requires=dbus.socket
 [Service]
 ExecStart=@EXPANDED_BINDIR@/dbus-daemon --session --address=systemd: --nofork --nopidfile --systemd-activation
 ExecReload=@EXPANDED_BINDIR@/dbus-send --print-reply --session --type=method_call --dest=org.freedesktop.DBus / org.freedesktop.DBus.ReloadConfig
-
-[Install]
-Also=dbus.socket
diff --git a/bus/systemd-user/dbus.socket.in b/bus/systemd-user/dbus.socket.in
index 4793d1d4ff1dc13e..304ac4034208f811 100644
--- a/bus/systemd-user/dbus.socket.in
+++ b/bus/systemd-user/dbus.socket.in
@@ -4,7 +4,3 @@ Description=D-Bus User Message Bus Socket
 [Socket]
 ListenStream=%t/bus
 ExecStartPost=-@SYSTEMCTL@ --user set-environment DBUS_SESSION_BUS_ADDRESS=unix:path=%t/bus
-
-[Install]
-WantedBy=sockets.target
-Also=dbus.service
-- 
2.9.3

